### ISSUE TITLE
Do security review on 215 petition page

### ISSUE THREAD

**Issue by OpenMatt** [MozillaFoundation/movement-building#27]
This is ready for sec review: https://github.com/MozillaFoundation/movement-building/issues/25

cc @simonwex @ScottDowne @davidascher

**Comment by OpenMatt** [comment]
They're starting QA on the page now.

**Comment by OpenMatt** [comment]
@ScottDowne has now completed initial QA and is waiting for final approval from Andrea
- @simonwex or @davidascher : please tap the mic on this ticket to let us know next steps

**Comment by davidascher** [comment]
Given that from a technology POV, as I understand it, this is mostly a copy & paste from the NN campaign, I'm not particularly worried.  I would suggest that @ScottDowne do a walk through of the non-layout diffs with someone.  @simonwex to figure out who.

**Comment by OpenMatt** [comment]
Cool! Thanks David. @ScottDowne + @simonwex : over to you guys to close this ticket when ready.

**Comment by ScottDowne** [comment]
Yeah, a security review should be quick. Anyone know who did the sec review on the NN petition?

**Comment by simonwex** [comment]
@ScottDowne could you book some time with me tomorrow to go over it? I'll do the review.

**Comment by ScottDowne** [comment]
@simonwex I booked some time you had free. Not sure if I booked it in time to be appropriate but I am flexible (and can do after 5 EST) so let me know if it's not ideal and we can find something better.

**Comment by OpenMatt** [comment]
@simonwex @ScottDowne what's the status on this one? can we mark closed?

**Comment by simonwex** [comment]
tl;dr: green light.

@OpenMatt the work specific to the petition page is good-to-go. There is still an open ticket in bugzilla where folks are looking for some potential BSD issues. -- If there are valid issues here, our exposure wouldn't be any different than EOY campaign or NN. Also our recourse wouldn't be fast enough for this project.

@ScottDowne could you link to the bugzilla ticket?

**Comment by ScottDowne** [comment]
@simonwex this one? https://bugzilla.mozilla.org/show_bug.cgi?id=1151659

If you're not cc'ed on it you won't be able to view it.

**Comment by ScottDowne** [comment]
Ah, likely this is the one you meant.

https://github.com/MozillaFoundation/movement-building/issues/36

**Comment by cassiemc** [comment]
@simonwex @ScottDowne can we close this?

**Comment by davidascher** [comment]
Or at least unassign me (which, funnily enough, I can't do :)
